doc/Extensions/RRDCached.md

```diff
@@ -89,7 +71,7 @@ WRITE_JITTER=1800
 BASE_PATH=/opt/librenms/rrd/
 JOURNAL_PATH=/var/lib/rrdcached/journal/
 PIDFILE=/run/rrdcached.pid
-SOCKFILE=/run/rrdcached.sock
+SOCKFILE=/var/run/rrdcached.sock
```

Comment:
/run/rrdcached.sock is the preferred path (remember /var/run is a symlink to /run`